dartagnan/src/main/java/com/dat3m/dartagnan/program/event/EventFactory.java

```diff
@@ -823,6 +823,10 @@ public static GenericVisibleEvent newAvDevice() {
         public static GenericVisibleEvent newVisDevice() {
             return new GenericVisibleEvent("visdevice", Tag.Vulkan.VISDEVICE);
         }
+
+        public static GenericVisibleEvent newUnreachable() {
+            return new GenericVisibleEvent("OpUnreachable", Tag.UB);
+        }
```

Comment:
If you add handling of this event into `stdlib.cat`, then I think the event should be part of the core section of the EventFactory (not SPIRV/Vulkan specific) and should be named "Unreachable" without the "Op"-prefix.
LLVM also has unreachable events btw., but I think we replace them by either `assert false` or `return` (not sure).